docs/io/fits/index.rst

```diff
@@ -547,6 +567,12 @@ numbers::
     >>> col1 = fits.Column(name='target', format='20A', array=a1)
     >>> col2 = fits.Column(name='V_mag', format='E', array=a2)
 
+.. note::
+
+    It is not necessary to create :class:`Column` object explicitly
+    if the data is stored in a
+    `structured array <https://docs.scipy.org/doc/numpy/user/basics.rec.html>`_.
+
```

Comment:
This PR is already merged but if you remember where it is, I can open another one to add it.